src/modules/main/controllers/admin-read.controller.ts

```diff
@@ -273,6 +289,14 @@ export class AdminReadController {
     return mapper.createWalletDetailDto(item);
   }
 
+  @Get('wallet/trial/:id')
+  async getWalletEvaDetail(@Param('id', ParseUUIDPipe) id: WalletId) {

```

Comment:
we already have endpoints to return wallets, moreover this endpoint is by no means bound to current user